tasks/sep/006-0-sc-changes/SignFromJson.s.sol

```diff
@@ -0,0 +1,523 @@
+        console.log("Running assertions on the DeputyGuardianModule");
+
+        require(deputyGuardianModule.version().eq("1.0.0"), "checkDeputyGuardianModule-000");
+        require(deputyGuardianModule.deputyGuardian() == address(foundationSafe), "checkDeputyGuardianModule-100");
+        require(deputyGuardianModule.safe() == address(securityCouncilSafe), "checkDeputyGuardianModule-200");
+        require(
+            deputyGuardianModule.superchainConfig() == address(proxies.SuperchainConfig),
+            "checkDeputyGuardianModule-300"
+        );
+    }
+
+    function checkSecurityCouncilSafe() internal view {
+        console.log("Running assertions on the SecurityCouncilSafe");
+
+        // The SecurityCouncilSafe and FoundationSafe should have the same set of owners
+        address[] memory councilOwners = securityCouncilSafe.getOwners();
+        address[] memory foundationOwners = foundationSafe.getOwners();
+        require(councilOwners.length == foundationOwners.length + 1, "checkSecurityCouncilSafe-200");
+        for (uint256 i = 0; i < councilOwners.length; i++) {
+            if (councilOwners[i] != deployerAddress) {
+                require(foundationSafe.isOwner(councilOwners[i]), "checkSecurityCouncilSafe-201");
+            }
+        }
+
+        // See sepolia.json for config values being verified:
+        // https://github.com/ethereum-optimism/optimism/pull/10224/files
+        // The SecurityCouncilSafe should have the same threshold (2) as the FoundationSafe
+        require(securityCouncilSafe.getThreshold() == foundationSafe.getThreshold(), "checkSecurityCouncilSafe-300");
+        require(securityCouncilSafe.getThreshold() == 2, "checkSecurityCouncilSafe-301");
+    }
+
+    /// @notice Checks the correctness of the deployment
+    function _postCheck(Vm.AccountAccess[] memory accesses, SimulationPayload memory /* simPayload */ )
+        internal
+        view
+        override
+    {
+        console.log("Running post-deploy assertions");
+
+        checkStateDiff(accesses);
+        checkSemvers();
+
+        checkSystemConfig();
+        checkL1CrossDomainMessenger();
+        checkL1StandardBridge();
+        checkL2OutputOracle();
+        checkOptimismMintableERC20Factory();
+        checkL1ERC721Bridge();
+        checkOptimismPortal();
+        checkProtocolVersions();
+        checkSuperchainConfig();
+        checkSecurityCouncilSafe();
+        checkLivenessModule();
+        checkLivenessGuard();
+        checkDeputyGuardianModule();
+
+        console.log("All assertions passed!");
+    }
+
+    function getCodeExceptions() internal view override returns (address[] memory) {
+        // Safe owners will appear in storage in the LivenessGuard when added
+        address[] memory securityCouncilSafeOwners = securityCouncilSafe.getOwners();
+        address[] memory shouldHaveCodeExceptions = new address[](7 + securityCouncilSafeOwners.length);
```

Comment:
6 because we have 6 hardcoded exceptions below (proposer, challenger, etc)? should the root `checkStateDiff` method in `JsonTxBuilderBase` check that no returned values are the zero address? 
```suggestion
        address[] memory shouldHaveCodeExceptions = new address[](6 + securityCouncilSafeOwners.length);
```